diuf/sudoku/Grid.java

```diff
@@ -77,9 +77,9 @@
     public static final int[][] cellRegions;
     public static final int[][] visibleCellIndex;
 	public static final int[][] forwardVisibleCellIndex;
-    private static final Block[] blocks;
-    private static final Row[] rows;
-    private static final Column[] columns;
+    public static final Block[] blocks;
+    public static final Row[] rows;
+    public static final Column[] columns;

```

Comment:
I intended to remove blocks,rows, columns. Now it is more difficult.
Are you using them in "explaining" part or also in "solving"?

Reply:
I'll look into this and reply

Reply:
The change was made due to this single line in both TurbotFish.java and ThreeStronLinks.java
```
return Grid.blocks[bridge1.getB()];
```

Reply:
privatized again!